app/javascript/packs/response-text.js

```diff
@@ -0,0 +1,46 @@
+window.addEventListener('load', () => {
+  function storeAllStoryTexts() {
+    $.get({
+      url: '/stories',
+      dataType: 'json',
+    }).done((res) => {
+      res.forEach((story) => {
+        if (story.data.attributes.response.length > 0) {
+          window.localStorage.setItem(`story${story.data.id}`, story.data.attributes.response.replace(/<p>|<\/p>/, ''));
+        }
+      });
+    });
+  }
+
+  function reloadStoryTexts() {
+    if (document.documentElement.clientWidth > 770) {
+      Array.from($('.story--response')).forEach((div) => {
+        if (div.children[0].children[0]) {
+          const modifiedDiv = div.children[0].children[0];
+          modifiedDiv.innerText = window.localStorage.getItem(`${div.id}`);
+        }
+      });
+    }
+  }
+
+  function lineClampResponseTexts() {
+    if (document.documentElement.clientWidth > 770) {
```

Comment:
Would this work better with one fo the breakpoints enumerated in our CSS file https://github.com/MAPC/digitalhub/blob/develop/app/assets/stylesheets/global/_vars.scss#L62-L66 or did this have to be 770 for a specific reason?

Reply:
That 770 is somewhat arbitrary, I changed it to 670, to correspond to `$break_small: 670px;`